src/modules/fancyzones/lib/FancyZones.cpp

```diff
@@ -630,7 +630,19 @@ void FancyZones::MoveWindowsOnDisplayChange() noexcept
 void FancyZones::UpdateDragState(require_write_lock) noexcept
 {
     const bool shift = GetAsyncKeyState(VK_SHIFT) & 0x8000;
-    m_dragEnabled = m_settings->GetSettings().shiftDrag ? shift : !shift;
+	const bool mouseR = GetAsyncKeyState(VK_RBUTTON) & 0x8000;
+	const bool mouseM = GetAsyncKeyState(VK_RBUTTON) & 0x8000;
+	const bool mouseX1 = GetAsyncKeyState(VK_XBUTTON1) & 0x8000;
+	const bool mouseX2 = GetAsyncKeyState(VK_XBUTTON2) & 0x8000;
+	const bool mouse =  mouseR | mouseM | mouseX1 | mouseX2;
+	bool enable = false;
+	if (m_settings->GetSettings().mouseDrag)
+		enable = enable | mouse;

```

Comment:
Use the curly braces even when there is only one line in the `if` block.
Thanks.

Reply:
I will make this change.